content/videos/challenges/139-calculating-digits-of-pi-with-collisions/index.json

```
@@ -0,0 +1,133 @@
+{
+    "title": "Calculating Digits of Pi with Collisions",
+    "description": "Happy Pi Day! In this video, I attempt to compute the digits of Pi using the \u201ccollisions\u201d method, thanks to 3Blue1Brown for the idea!",
+    "videoNumber": "139",
+    "videoId": "PoW8g67XNxA",
+    "date": "2019-03-14",
+    "languages": [
+        "p5.js",
+        "JavaScript"
+    ],
+    "topics": [
+        "3blue1brown",
+        "elastic collision",
+        "pi day",
+        "euler method"
+    ],
+    "canContribute": true,
+    "relatedChallenges": [
+        "95-approximating-the-value-of-pi",
+        "140-pi-leibniz",
+        "156-peeking-inside-pi",
+        "161-estimating-pi-from-random-numbers"
```

I honestly love both the thumbnails! I have requested a review from Dan, about the problem in the web editor and the thumbnail as well.

Also, I think another `relatedChallenge` could be added
```suggestion
        "161-estimating-pi-from-random-numbers",
        "169-pi-in-the-sky"
```